roles/confluent.test/molecule/plain-rhel/verify.yml

```diff
@@ -11,6 +11,24 @@
         property: listener.name.internal.sasl.enabled.mechanisms
         expected_value: PLAIN
 
+    # Secrets Protection
+    - import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+          file_path: /etc/kafka/server.properties
+          property: advertised.listeners
+          expected_value: ${securepass:./security.properties:server.properties/advertised.listeners}
```

Comment:
im a little curious what this path is relative to "./security.properties" but i suppose if it works it works